modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java

```diff
@@ -985,6 +989,76 @@ public void testAddObjectAction() throws Exception {
 		_objectActionLocalService.deleteObjectAction(systemObjectAction);
 	}
 
+	@Test
+	public void testAddObjectActionInSystemObjectDefinitionWithConditionExpression()
+		throws Exception {
+
+		SystemObjectDefinitionManager userSystemObjectDefinitionManager =
+			_systemObjectDefinitionManagerRegistry.
+				getSystemObjectDefinitionManager("User");
+
```

Comment:
@JhossephG please declare the variable as close as possible to its line of first usage (see this [article](https://liferay.atlassian.net/wiki/spaces/ENGR/pages/1511752519/Java+conventions+in+Liferay#[inlineExtension]Variables-are-only-declared-as-used,-not-before-(%E2%80%9CAs-used%E2%80%9D)))